sbt-plugin/src/main/scala/ch/epfl/scala/GithubDependencyGraphPlugin.scala

```diff
@@ -106,6 +106,7 @@ object GithubDependencyGraphPlugin extends AutoPlugin {
     val scalaBinaryVersion = (Keys.artifactName / Keys.scalaBinaryVersion).value
     val crossVersion = CrossVersion.apply(scalaVersion, scalaBinaryVersion)
     val allDirectDependencies = Keys.allDependencies.value
+    val ignoredDependencies = Keys.state.value.attributes(githubSubmitInputKey).ignoredDependencies
```

Comment:
The `githubSubmitInputKey` attribute is set by the `githubSubmitDependencyGraph` command [here](https://github.com/scalacenter/sbt-dependency-submission/blob/ddf9437e7861a7257cada2f6cbac06173150dd8e/sbt-plugin/src/main/scala/ch/epfl/scala/SubmitDependencyGraph.scala#L59).

I suggest to add a default value, for the task not to task if the attribute is missing:
```suggestion
    val ignoredDependencies = Keys.state.value.attributes.get(githubSubmitInputKey).toSeq.flatMap(_.ignoredDependencies)
```